include/seqan/basic/iterator_adapt_std.h

```diff
@@ -55,6 +55,17 @@ namespace std
         typedef typename seqan::Value<TIter>::Type * pointer; // nolint
         typedef typename seqan::Reference<TIter>::Type reference; // nolint
     };
+
+	// there is a bug in vc2015 stl, it doesnt check the iterator_traits correctly
+	// I have reported this bug to microsoft already (filed as DevDiv#1208117). 
+	// For now, this is a workaround.
+#if _MSC_VER == 1900
+	template<class _Ty, class Tag>
+	struct _Is_iterator<typename seqan::Iter<_Ty, Tag>>
```

Comment:
okay i will do next time, but this code is only active in vs2015 - i will change it anyway to make it more consistent :)